xbmc/dialogs/GUIDialogVolumeBar.cpp

```diff
@@ -27,7 +28,9 @@ CGUIDialogVolumeBar::CGUIDialogVolumeBar(void)
   : CGUIDialog(WINDOW_DIALOG_VOLUME_BAR, "DialogVolumeBar.xml", DialogModalityType::MODELESS)
 {
   m_loadType = LOAD_ON_GUI_INIT;
-  SetAutoClose(VOLUME_BAR_DISPLAY_TIME);
+
+  if (g_application.IsMuted() || g_application.GetVolume(false) <= VOLUME_MINIMUM)
+    Open();
```

Comment:
To open the dialog on startup we should place the check and the activation to https://github.com/xbmc/xbmc/blob/74ada972bfabc63102be7442775d75e5c4a198d1/xbmc/Application.cpp#L4181